.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+**/* @wleklinskimateusz @irosikoni
```

Comment:
this file can tell GitHub who should approve which part of the application, for now, I've given ownership to whole application to me and @irosikoni but in the future when work is more divided, we will split responsibilities a bit more